index.bs

```diff
@@ -4130,6 +4287,47 @@ follows:</p>
 </ol>
 
 
+<p>The <dfn>WebVTT attributes key/value parsing rules</dfn> consist of the following algorithm.</p>
+
+<ol algorithm="WebVTT attributes block parsing">
+ <li>Let |input| be the list of key/value pairs from a <a>WebVTT attributes block</a>.</li>
+ <li>
+  How the attribute is processed depends on its key name, as follows:
+  <dl>
+
+   <dt>If the key name is "<code>kind</code>"</dt>
+   <dd>Process the value as <a href="https://html.spec.whatwg.org/multipage/media.html#attr-track-kind">the kind attribute</a> of a track element according to the HTML Standard.</dd>
+
+   <dt>If the key name is "<code>srclang</code>"</dt>
+   <dd>Process the value as <a href="https://html.spec.whatwg.org/multipage/media.html#attr-track-srclang">the srclang attribute</a> of a track element according to the HTML Standard.</dd>
+
+   <dt>If the key name is "<code>label</code>"</dt>
+   <dd>Process the value as <a href="https://html.spec.whatwg.org/multipage/media.html#attr-track-label">the label attribute</a> of a track element according to the HTML Standard.</dd>
+
+   <dt>If the key name is "<code>type</code>" (TODO: For clarity, should this be "subkind" or "kind_subtype" instead?)</dt>
+   <dd>Process the value according to the <a>WebVTT type attribute parsing rules</a>.
+
+   <dt>Otherwise</dt>
+   <dd>Ignore the key/value pair.</dd>
+
+  </dl>
+ </li>
+</ol>
+
+<p>The <dfn>WebVTT type attribute parsing rules</dfn> consist of the following algorithm.</p>
+
+<ol algorithm="WebVTT type attribute attribute parsing">
+ <li>TODO: This could reference a new TBD W3C Note or Evergreen list of acknowledged kind subtypes, along with a reference to the specification for each, which clarify the usage or define further parsing rules of each type. For example:
+  <ul>
+   <li>metadata subtype: time-coded video poster thumbnails (common de facto use for scrubbing but no spec)</li>
+   <li>metadata subtype: <a href="https://github.com/w3c/webvtt/issues/512">WebVTT Issue 512: time-coded flash metadata</a></li>
+   <li>caption or description subtype: text equivalent of audio description audio track (used for braille displays)</li>
+   <li>etc.</li>
+  </ul>
+ </li>
+</ol>
```

Comment:
This whole section is effectively a TODO that is not required for #511 but shows the potential future direction of clarifying the `type` of metadata or other kinds of VTT. Feedback requested.

Reply:
Another instance of this disambiguation need outlined in:
- https://github.com/w3ctag/design-reviews/issues/952